spec/abstractops.html

```diff
@@ -323,11 +323,20 @@ <h1>ToSecondsStringPrecision ( _normalizedOptions_ )</h1>
     </emu-alg>
   </emu-clause>
 
-  <emu-clause id="sec-temporal-gettemporalunit" aoid="GetTemporalUnit">
-    <h1>GetTemporalUnit ( _normalizedOptions_, _key_, _unitGroup_, _defaultValue_ [ , _extraValues_ ] )</h1>
-    <p>
-      The abstract operation GetTemporalUnit attempts to read from the specified property of the Object _normalizedOptions_ a Temporal unit that is covered by the union of _unitGroup_ (~date~, ~time~, or ~datetime~) and _extraValues_, substituting _defaultValue_ if the property value is *undefined*.
-    </p>
+  <emu-clause id="sec-temporal-gettemporalunit" type="abstract operation">
+    <h1>
+      GetTemporalUnit (
+        _normalizedOptions_: an Object,
+        _key_: a property key,
```

Comment:
Optional, we could narrow this down to "a String" here because options bags with integer/symbol keys are probably not what we want :smile:

Reply:
Fair, but I think that's best handled at invocation sites rather than being an unnecessary built-in constraint of the operation.